apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ejs

```diff
@@ -44,14 +44,14 @@
                                 <th id="earliest-access-date">
                                     Earliest Access Date
                                     <a role="button" class="btn btn-xs btn-light" data-placement="auto" data-trigger="focus" data-toggle="popover" data-html="true" title="Order By" tabindex="0"
-                                        data-content="<%= include('popoverStartDate'); %>">
+                                        data-content="<%= include('popoverStartDate'); %>" aria-label="Help on Earliest Access Date">
```

Comment:
As long as you're here: the `title` here doesn't seem relevant to this column. Could you update it to "Earliest Access Date"? Ditto for "Latest Access Date" below.

"Help on ..." reads a little awkwardly to me. Consider "Information about Earliest Access Date" instead; how do you feel about that?

Reply:
Totally fine, all good! Making changes....

Thanks!

Reply:
Ohhh yes right that title! Order by! Yes, I had noticed that on the web, and then got too distracted by the second part of your comment. Adjusting that title now!